com.unity.visualeffectgraph/Runtime/Utilities/PropertyBinding/Implementation/VFXInputAxisBinder.cs

```diff
@@ -22,6 +22,7 @@ public override bool IsValid(VisualEffect component)
 
         public override void UpdateBinding(VisualEffect component)
         {
+#if ENABLE_LEGACY_INPUT_MANAGER
```

Comment:
How does this behave when com.unity.input *isn't* installed ?
Is ENABLE_LEGACY_INPUT_MANAGER=1 automatically defined ?
Otherwise, it should be 
#if ENABLE_LEGACY_INPUT_MANAGER || !ENABLE_INPUT_SYSTEM_PACKAGE

Reply:
This defined is supposed to be managed by Unity: https://docs.unity3d.com/Manual/PlatformDependentCompilation.html
So it should work disable the define if the module is not installed